integration-tests/itest/go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/lightbend/cloudflow/integration-test/itest
 go 1.14
 
 require (
-	github.com/onsi/ginkgo v1.12.0
-	github.com/onsi/gomega v1.9.0
+	github.com/onsi/ginkgo v1.12.3
+	github.com/onsi/gomega v1.10.1
```

Comment:
Looks like doing a `go get` automatically updated these for me. And they seem to be working. So I kept the updated versions.